lambdas/jest.config.ts

```diff
@@ -9,10 +9,10 @@ export default {
     setupFiles: ["<rootDir>/setEnvVars.js"],
     coverageThreshold: {
         global: {
-            statements: 95,
-            branches: 95,
+            statements: 94,
```

Comment:
This change shouldn't be needed anymore, now that you've removed the code changes.

Reply:
It is still needed - I've tested just now and jest fails with the old thresholds - see [this run](https://github.com/govuk-one-login/ipv-cri-common-lambdas/actions/runs/7501550989/job/20422416611)

Reply:
Ah damn. Ok.